build.gradle

```diff
@@ -0,0 +1,92 @@
+plugins {
```

Comment:
Please don't just change the build system to gradle. This kind of change has basically no value and is impossible to review when combined with other codebase changes.